application/controllers/Campaign.php

```diff
@@ -561,6 +561,8 @@ public function status($id = null, $component = null, $selected_milestone = null
             $this->db->where('cfo_act_agency', 'true');
         } else if ($id == 'omb-monitored') {
             $this->db->where('omb_monitored', 'true');
```

Comment:
`omb_monitored` and `long_running` aren't really orthogonal to `long_running`, so this if-else feels off... There can be `cfo-act` and/or `omb-monitored` agencies that are also `long_running`;`long_running` is a subset of the rest. 

This logic also enables running a crawl for only those agencies which are `long-running`, but how would you enable specifying a crawl that _excludes_ long-running agencies? It might be better to just explicitly leave long-running agencies _out_ of the `omb-monitored` and `cfo-act` selection and document that. In other words, those options would only select agencies where `long_running` is false, and long-running agencies only ever get crawled if you specifically ask for it.)

Reply:
Adding one more line to the first two conditional branches stating that `long_running` is false